projects/storefrontlib/src/cms-components/myaccount/my-interests/my-interests.component.spec.ts

```diff
@@ -86,180 +88,137 @@ const mockedInterests: ProductInterestList = {
         },
       ],
     },
+    {
+      product: {
+        code: '553638',
+        name: 'NV11',
+        images: {
+          PRIMARY: {
+            thumbnail: {
+              altText: 'NV11',
+              format: 'thumbnail',
+              imageType: ImageType.PRIMARY,
+              url: 'image-url',
+            },
+          },
+        },
+        price: {
+          formattedValue: '$188.69',
+        },
+        stock: {
+          stockLevel: 0,
+          stockLevelStatus: 'outOfStock',
+        },
+      },
+      productInterestEntry: [
+        {
+          dateAdded: new Date(),
+          interestType: 'BACK_IN_STOCK',
+        },
+      ],
+    },
   ],
 };
+const emptyInterests: ProductInterestList = {
+  sorts: [{ code: 'name', asc: true }],
+  pagination: {},
+};
+const userToken = {
+  access_token: 'xxx',
+  token_type: 'bearer',
+  refresh_token: 'xxx',
+  expires_in: 1000,
+  scope: ['xxx'],
+  userId: 'xxx',
+};
 
 describe('MyInterestsComponent', () => {
   let component: MyInterestsComponent;
   let fixture: ComponentFixture<MyInterestsComponent>;
-  let userService: UserService;
   let el: DebugElement;
+  const userService = jasmine.createSpyObj('UserService', [
+    'loadProductInterests',
+    'getProdutInterests',
+    'getProdutInterestsLoaded',
+    'deleteProdutInterest',
+    'clearProductInterests',
+  ]);
+  const authService = jasmine.createSpyObj('AuthService', ['getUserToken']);
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [
-        RouterTestingModule,
-        ListNavigationModule,
-        I18nTestingModule,
-        MediaModule,
-      ],
+      imports: [RouterTestingModule, ListNavigationModule, I18nTestingModule],
       providers: [
-        { provide: UserService, useClass: MockUserService },
-        { provide: AuthService, useClass: MockAuthService },
+        { provide: UserService, useValue: userService },
+        { provide: AuthService, useValue: authService },
+        { provide: OccConfig, useValue: MockOccModuleConfig },
+        { provide: LayoutConfig, useValue: MockLayoutConfig },
```

Comment:
As we discussed, you need to import media module rather than inject LayoutConfig.